git-date/src/parse.rs

```diff
@@ -1,11 +1,50 @@
+use crate::time::format::{RFC2822, SHORT};
 use crate::Time;
+use time::{Date, OffsetDateTime};
 
 #[allow(missing_docs)]
 pub fn parse(input: &str) -> Option<Time> {
```

Comment:
Before I forget, the current time should be passed here. That way tests are deterministic.